### ISSUE TITLE
an whitelist station Option that helps in -wP	 a lot !!!!!

### ISSUE THREAD

**Issue by minanagehsalalma** [wifiphisher/wifiphisher#1083]
https://github.com/aircrack-ng/mdk4/issues/26 

instead of disconnecting all the clients in the -wP scenario..... we disconnecting all  the clients expect one (ours) the -wAI  --wpspbc-assoc-interface
...
so we no longer need the stop the DoS attacks when the user clicks the i pressed the WPS PBC button !

**Comment by minanagehsalalma** [comment]
@sophron

**Comment by djdisikd** [comment]
Good idea but I don't think many people knew what is WPS PBC, all they knew was the password

**Comment by minanagehsalalma** [comment]
@djdisikd  

> but I don't think many people knew what is WPS PBC, all they knew was the password

lol why is that !! the youtube -facebook-playstore full of wps applications / tutorials 
once they check that there APs aren't  vulnerability .. they will keep it on and learn how it works ..
and we have already discussed this here https://github.com/wifiphisher/wifiphisher/issues/494
